WebRoot/css/zm.css

```diff
@@ -3807,6 +3809,37 @@ HTML>BODY .appt-selected .appt_allday_body
 	padding-right: 0;
 }
 
+/* Out-of-Office preference section */
+.ZmOutOfOfficeWrapper {
+	@PrefOOOWrapper@
```

Comment:
I think it's no harm in keeping that, that property enables to override wrapper style in some custom skin.

We can remove it though. Let me know.

Thanks